specs/trip_dispatch_spec.rb

```diff
@@ -89,4 +81,41 @@
       passenger.trips.must_include trip
     end
   end
+
+  describe "Driver Status" do
+    it 'checks the status of a driver' do
+      # Arrange
+      dispatcher = RideShare::TripDispatcher.new
+      # Act
+      driver = dispatcher.select_available_driver
+      # Assert
+      driver.id.must_equal 2
+      driver.status.must_equal :AVAILABLE
+    end
+  end
+
+  describe 'Request Trip' do
+    it 'will raise an ArgumentError when nothing is entered for Passenger ID' do
```

Comment:
What if there is no available driver? This is an important case, and is exactly the sort of situation where software will do something unexpected. You might set up the test as follows:

```ruby
it 'raises an error if no driver is available' do
  @dispatcher.drivers.each do |driver|
    # This may require an attr_writer in Driver
    driver.status = :UNAVAILABLE
  end
  proc {
    @dispatcher.request_trip(2)
  }.must_raise StandardError
end
```